app/models/ingest_job.rb

```diff
@@ -389,6 +390,17 @@ def set_study_default_options
     self.study.save
   end
 
+  # set the point count on a cluster group after successful ingest
+  #
+  # * *yields*
+  #   - sets the :points attribute on a ClusterGroup
+  def set_cluster_point_count
+    cluster_group = ClusterGroup.find_by(study_id: self.study.id, study_file_id: self.study_file.id)
+    Rails.logger.info "Setting point count on #{cluster_group.name}:#{cluster_group.id}"
+    cluster_group.set_point_count!
+    Rails.logger.info "Point count on #{cluster_group.name}:#{cluster_group.id} set to #{cluster_group.points}"
```

Comment:
Do we need both of these statements?  I think we just need the second one -- since set_point_count throws exception on error, we'll know if it fails.

Reply:
That's a great point - I can change that.